packages/react-components/src/index.ts

```diff
@@ -216,5 +216,10 @@ export {
   WorkingGroupAbout,
 } from './templates';
 export * from './theme';
-export { noop } from './utils';
+export {
+  noop,
+  isDirtyEditMode,
```

Comment:
This seems short sighted if we need to have a Teams edit more and a Working Group Edit mode. @kmaid figure out whats going on here